support-frontend/assets/components/packshots/paper-packshot.jsx

```diff
@@ -5,31 +5,13 @@ import GridImage from 'components/gridImage/gridImage';
 
 const PaperPackshot = () => (
   <div className="subscriptions__paper-packshot">
-    <GridImage
-      gridId="subscriptionFeast"
-      srcSizes={[660, 500, 140]}
-      sizes="(max-width: 740px) 100%,
-             (max-width: 980px) 178px,
-             (max-width: 1140px) 198px,
-             220px"
-      imgType="png"
-    />
     <GridImage
       gridId="subscriptionPrint"
-      srcSizes={[805, 402]}
-      sizes="(max-width: 740px) 100%,
-             (max-width: 980px) 240px,
-             (max-width: 1140px) 311px,
-             346px"
-      imgType="png"
-    />
-    <GridImage
-      gridId="subscriptionG2"
-      srcSizes={[756, 500, 140]}
-      sizes="(max-width: 740px) 100%,
-             (max-width: 980px) 204px,
-             (max-width: 1140px) 227px,
-             252px"
+      srcSizes={[500, 140]}
+      sizes="(max-width: 739px) 140px,
+             (max-width: 979px) 500px,
+             (max-width: 1140px) 500px,
+             500px"
```

Comment:
Is the 1px difference in some of the media-queries here necessary?

Reply:
Here I'd followed the convention in the codebase but I do believe the 1px difference may not be necessary due to cascading

Reply:
OK great, if the previous implementation went against the convention then we should retain the changes you've made.